scripts/README.md

```diff
@@ -15,6 +17,17 @@ Run the following script to provision the infrastructure and deploy the API and
                              -subscription <subscription-id>
 ```
 
+## Using ACA
+
+Deployment using ACA instead of AKS requires the addition of the argument `-deployAks 0` to the command line call.
+
+```pwsh
+./scripts/Unified-Deploy.ps1 -resourceGroup <resource-group-name> `
+                             -location <location> `
+                             -subscription <subscription-id> `
+                             -deployAks 0
+```
+
 ## Deployments using an existing OpenAI service
 
 For deployments that need to use an existing OpenAI service, run the following from the `scripts`.  This will provision all of the necessary infrastruction except the Azure OpenAI service and will deploy the API and frontend to an AKS cluster via Helm.
```

Comment:
Where do you enter the account key for OpenAI?

Reply:
That is obtained from the OpenAI instance created by the ARM template deployment.  I am currently working on adding back support for pre-existing OpenAI instances, so the argument for explicitly setting the OpenAI account key along with the other OpenAI specific arguments will be present in a later commit soon.